src/applications/vaos/tests/mocks/v2.js

```diff
@@ -41,12 +40,48 @@ export function getVAOSRequestMock() {
       serviceType: null,
       slot: null,
       start: null,
-      status: null,
+      status: 'proposed',
       telehealth: null,
     },
   };
 }
 
+/**
+ * Return a stubbed VAOS VA appointment item
+ *
+ * @export
+ * @returns {VAOSRequest} VAOS VA request object
+ */
+export function getVAOSAppointmentMock() {
```

Comment:
Needed to create a separate default mock function for requests and appointments, even though the differences are pretty subtle.